03-Custom-Actions-In-Python/04-Custom-Updater.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>This question seems a bit specific. Maybe something more like "How can I modify the state of the system in a custom updater?"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>